package.json

```diff
@@ -59,11 +59,6 @@
         "preset": "ts-jest/presets/default-esm",
         "testEnvironment": "node",
         "coverageProvider": "v8",
-        "globals": {
```

Comment:
Unrelated change, but this removes a deprecation warning from `ts-jest`. There is no change in behavior.